core/src/config/project.ts

```diff
@@ -283,7 +283,7 @@ export const projectSchema = createSchema({
     "Configuration for a Garden project. This should be specified in the garden.yml file in your project root.",
   required: true,
   keys: () => ({
-    apiVersion: apiVersionSchemaWithoutDefault().required(),
+    apiVersion: apiVersionSchema().description("The schema version of the config."),
```

Comment:
```suggestion
    apiVersion: apiVersionSchema().description("Schema version of the config."),
```

I think it flows better without the "the". Also, if you change the in-code doc new reference documentation has to be rendered. You can do so by running `yarn build` in the git root.